pkg/user/user_integration_test.go

```diff
@@ -303,6 +591,30 @@ func TestUserHandler(t *testing.T) {
 	})
 }
 
+type userService interface {
+	FindById(context context.Context, id uint) (*model.User, error)
+	ValidateEmail(emailToken uuid.UUID) error
+}
+
+func createUser(t *testing.T, client *inttest.HTTPClient, userService userService, email string, password string) *model.User {
```

Comment:
call https://pkg.go.dev/testing#T.Helper in helpers so we get better error messages